src/serum-version.h

```diff
@@ -2,7 +2,7 @@
 
 #define SERUM_VERSION_MAJOR 1 // X Digits
 #define SERUM_VERSION_MINOR 3  // Max 2 Digits
-#define SERUM_VERSION_PATCH 1  // Max 2 Digits
```

Comment:
1.3.1 is not released yet

Reply:
okay, yeh, I got confused because of the defect where it generated "1.3." instead of "1.3.0"

Reply:
Removed bumped version (ie, it's still 1.3.1)